src/Sylius/Bundle/AddressingBundle/Form/EventListener/BuildAddressFormSubscriber.php

```diff
@@ -137,6 +137,8 @@ private function createProvinceCodeChoiceForm(CountryInterface $country, $provin
         return $this->formFactory->createNamed('provinceCode', ProvinceCodeChoiceType::class, $provinceCode, [
             'country' => $country,
             'auto_initialize' => false,
+            'label' => 'sylius.form.address.province',
```

Comment:
@pamil I had only extended it to the dropdown as well because I noticed they hadn't been defined in there either when they had been in the class handling realtime province field updates. The dropdown part is untested, only assumed. My issue is indeed fixed just by adjusting `createProvinceNameTextForm`